packages/chain-bridge/test/mocks/MockAdapter.sol

```diff
@@ -144,7 +145,7 @@ contract MockAdapter is IBridgeAdapter {
         BridgeTypes.AdapterParams calldata
     ) external payable {
         // Check caller is bridge router
-        if (msg.sender != bridgeRouter) revert Unauthorized();
+        if (msg.sender != bridgeRouter) revert BaseBridgeAdapter.Unauthorized();
```

Comment:
seems that the MockAdapter will be hard to maintain during the dev phase, maybe we shall move to a similar approach as with the `Ark.sol` and implement all the security features in the base contract - then override where required in the specifica adapters.